server/auth.go

```diff
@@ -584,9 +586,15 @@ func (s *Server) processClientOrLeafAuthentication(c *client, opts *Options) boo
 			return false
 		}
 		issuer := juc.Issuer
-		if juc.IssuerAccount != "" {
+		if juc.IssuerAccount != _EMPTY_ {
 			issuer = juc.IssuerAccount
 		}
+		if pinnedAcounts != nil {
+			if _, ok := pinnedAcounts[issuer]; !ok {
+				c.Debugf("Account not listed as operator pinned account")
```

Comment:
Do we have varz metrics we can bump up here?  I'd like it if "tried to authenticate as an unpinned account" bumped up the counter.

Also, what is our threat-model around enumeration of "which accounts are allowed to connect to this server", given public operators with public account lists?  Should we move the rejection to _after_ otherwise-successful authentication, to avoid timing-attacks based on differences in timing?

Reply:
added the counter